From aa9981bcf4f02bdcedb3b7df78056630ca564125 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Fri, 6 Jun 2025 15:35:06 +0000
Subject: [PATCH] Fix for Fuzzer error with 'lunch caas-ap3a-user'

When giving full build with lunch caas-ap3a-user
we get error that the newly added SharedSecret
service has no fuzzer entry

Added the required fuzzer entry

Tests Done:
1. Flash the image Bare metal or CiV
2. Image is booting and service is running

Tracked-On: OAM-123515
Change-Id: I304a4371e821c829f979e3033df95c04eaf01d5d
Signed-off-by: Bhadouria, Aman <aman.bhadouria@intel.com>
---
 build/soong/service_fuzzer_bindings.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/soong/service_fuzzer_bindings.go b/build/soong/service_fuzzer_bindings.go
index dd83ae0e8..d81137bf5 100644
--- a/build/soong/service_fuzzer_bindings.go
+++ b/build/soong/service_fuzzer_bindings.go
@@ -147,6 +147,7 @@ var (
 		"android.hardware.security.sharedsecret.ISharedSecret/default":                       EXCEPTION_NO_FUZZER,
 		"android.hardware.security.sharedsecret.ISharedSecret/hwcrypto":                      EXCEPTION_NO_FUZZER,
 		"android.hardware.security.sharedsecret.ISharedSecret/security_vm":                   EXCEPTION_NO_FUZZER,
+		"android.hardware.security.sharedsecret.ISharedSecret/gatekeeper":                    EXCEPTION_NO_FUZZER,
 		"android.hardware.security.timestamp.ITimeStamper/default":                           EXCEPTION_NO_FUZZER,
 		"android.hardware.sensors.ISensors/default":                                          EXCEPTION_NO_FUZZER,
 		"android.hardware.soundtrigger3.ISoundTriggerHw/default":                             EXCEPTION_NO_FUZZER,
-- 
2.34.1

